platform/linux-generic/pool/subsystem.c

```diff
@@ -15,3 +19,219 @@ ODP_SUBSYSTEM_CONSTRUCTOR(pool)
 	odp_subsystem_constructor(pool);
 }
 
+int odp_pool_init_global(void)
+{
+	pool_module_t *mod;
+
+	mod = odp_subsystem_active_module(pool, mod);
+	if (mod == NULL) {
+		ODP_ERR("No active module in pool subsystem\n");
+		return -1;
+	}
+	if (mod->base.init_global == NULL) {
+		ODP_ERR("No defined init_global function "
+			"in module %s of pool subsystem\n", mod->base.name);
+		return -1;
+	}
```

Comment:
Yes, we saw old code also does not check NULL pointer errors in API wrappers, here can do the same and assume all modules understand this requirement.